.travis.yml

```diff
@@ -6,6 +6,7 @@ install:
   - pip install -r requirements.txt
   - pip install coveralls
 script:
-  coverage run --source=amy,api,workshops manage.py test
+  - python manage.py makemigrations --dry-run -e; export STATUS_CODE=$?; if [[ "$STATUS_CODE" == "0" ]]; then exit 1; fi
```

Comment:
Well, if there are some missing migrations in other applications, that indicates that something is wrong. Am I right that it should never happen that we have a 3rd party app (or django app) with missing migrations?